app/models/state_file_md_intake.rb

```diff
@@ -231,6 +231,17 @@ def sum_1099_r_followup_type_for_filer(primary_or_spouse, followup_type)
     end
   end
 
+  def sum_two_1099_r_followup_types_for_filer(primary_or_spouse, followup_type_1, followup_type_2)
```

Comment:
major nit but i think this would be clearer if the arguments were named like this instead:
`followup_type_1` -> `income_source`
`followup_type_2 ` -> `service_type`

Reply:
🤔 ya i wanted to make this method re-usable for other follow up types, so that's why the generic-ish name. but, i can do that.